app/src/main/java/org/wikipedia/usercontrib/UserInformationDialog.kt

```diff
@@ -72,6 +73,7 @@ class UserInformationDialog : DialogFragment() {
     }
 
     private fun onSuccess(editCount: String, registrationDate: Date) {
+        PatrollerExperienceEvent.logAction("user_info_impression", "pt_recent_changes")
```

Comment:
Same concern as above. Should we log this if the source is from patroller tasks?